tgui/packages/tgui/interfaces/CrewCredits.tsx

```diff
@@ -0,0 +1,75 @@
+import { BooleanLike } from "common/react";
+import { useBackend } from "../backend";
+import { Box, Section } from "../components";
+import { Window } from "../layouts";
+
+interface CrewCreditsData {
+  darkMode: BooleanLike,
+  groups: GroupBlockProps[]
+}
+
+export const CrewCredits = (props, context) => {
+  const { data } = useBackend<CrewCreditsData>(context);
+  return (
+    <Window title="Crew Credits" width={600} height={600} theme={data["darkMode"] ? "neutral" : "paper"}>
+      <Window.Content scrollable>
+        {data.groups?.map((group, index) =>
+          (!!(data.groups[index].crew.length > 0) && <GroupBlock key={index} group={group.group} crew={group.crew} />)
```

Comment:
Already a boolean check.
```suggestion
          ((data.groups[index].crew.length > 0) && <GroupBlock key={index} group={group.group} crew={group.crew} />)
```